index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Find Code here</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="code">
+        <h2>What is COD_?</h2>
+    </div>
+    <div class="container">
+      
+        <div class="text" style="--j:0;">
+            <span style="--i:0">C</span>
+            <span style="--i:1">3</span>
+            <span style="--i:2">4</span>
+            <span style="--i:3">5</span>
+
+        </div> 
+         <div class="text" style="--j:1;">
+            <span style="--i:0">O</span>
+            <span style="--i:1">1</span>
+            <span style="--i:2">2</span>
+            <span style="--i:3">3</span>
+
+        </div>
+        <div class="text" style="--j:2;">
+            <span style="--i:0">D</span>
+            <span style="--i:1">3</span>
+            <span style="--i:2">4</span>
```

Comment:
Its better to use letter in place of numbers, as your main content is of letters